Tests/SQLiteTests/TestHelpers.swift

```diff
@@ -125,4 +125,15 @@ class TestCodable: Codable {
         self.optional = optional
         self.sub = sub
     }
+    
+    static func == (lhs: TestCodable, rhs: TestCodable) -> Bool {
```

Comment:
the compiler can generate this for you

Reply:
Works only for structs, not classes. And this class cannot be a struct because it contains a nested object with same class.